go/fn/examples/example_asmain_runner_test.go

```diff
@@ -4,7 +4,7 @@
 // you may not use this file except in compliance with the License.
 // You may obtain a copy of the License at
 //
-//      http://www.apache.org/licenses/LICENSE-2.0
+//	http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
nit: What caused the license header change?

Reply:
Good catch. I split my changes to several PRs. This is introduced from copy-paste in GitHub "Raw" mode.